app/errors.js

```diff
@@ -6,10 +6,10 @@ const internalError = (message, internalCode) => ({
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
 
-// exports.userNotFound = {
-//   message: 'The user was not found.',
-//   internalCode: 'user_not_found'
-// };
+exports.albumAlreadyPurchased = {
```

Comment:
remember you could use the internalError function and create some more customizable messages like using params like the id.